dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -167,13 +169,15 @@ const TableWithFavorite = () => {
             isSelected={isSelected === "datasetListButton"}
             onChange={handleButtonClick}
             className="datasetListButton"
+            aria-label="see dataset button"
           />
           <ToggleGroupItem
             text={`Favorites(${favoriteRepoNames?.length})`}
             buttonId="favoriteListButton"
             isSelected={isSelected === "favoriteListButton"}
             onChange={handleButtonClick}
             className="favoriteListButton"
+            aria-label="see favorites button"
```

Comment:
While this is clever, and including the accessibility labels is probably good anyway, I'm wondering if most or all of this targeting couldn't be done with the more common `buttonId` tags??

Reply:
> that failed

How did it fail?  Did it find no element, or did it find more than one?  (Or, was there some other objection?)

Reply:
`TestingLibraryElementError: Unable to find an accessible element with the role "button" and name "favoriteListButton"`

Reply:
A brief bit of research suggests that `getByRole()` is probably the right choice of function, and specifying `"button"` is appropriate, and that the remaining options target a number of characteristics which do _not_ include the ID.  The targeting seems to have a much more user-focused perspective, so they key off items which are user-accessible, including the ARIA values.  So, I think Shubham's on the right track, here.